webapp/src/main/java/ar/edu/itba/paw/webapp/controllers/old/OldStudentController.java

```diff
@@ -0,0 +1,783 @@
+//			LOGGER.warn("User {} could not unenroll docket {} due to {} [POST]", loggedUser.getDni(), docket, errors.getAllErrors());
+//			redirectAttributes.addFlashAttribute("org.springframework.validation.BindingResult.inscriptionForm", errors);
+//			redirectAttributes.addFlashAttribute("inscriptionForm", inscriptionForm);
+//			return new ModelAndView("redirect:/students/" + docket + "/courses");
+//		}
+//
+//		final boolean done = studentService.unenroll(inscriptionForm.getStudentDocket(), inscriptionForm.getCourseId());
+//
+//		if (!done) {
+//			LOGGER.warn("User {} could not unenroll a student {}, Result = {}", loggedUser.getDni(), docket, done);
+//			redirectAttributes.addFlashAttribute("alert", "danger");
+//			redirectAttributes.addFlashAttribute("message", messageSource.getMessage("ERROR_UNKNOWN", null, Locale.getDefault()));
+//		} else {
+//			LOGGER.info("User {} unenrolled student {} successfully", loggedUser.getDni(), docket);
+//			redirectAttributes.addFlashAttribute("alert", "success");
+//			redirectAttributes.addFlashAttribute("message", messageSource.getMessage("unenroll_success",
+//					new Object[] {inscriptionForm.getCourseId(), inscriptionForm.getCourseName()},
+//					Locale.getDefault()));
+//		}
+//
+//		return new ModelAndView("redirect:/students/" + docket + "/courses");
+//
+//	}
+
+	@RequestMapping(value = "/students/{docket}/inscription", method = RequestMethod.GET)
+	public ModelAndView studentInscription(@PathVariable final int docket, final Model model,
+	                                       @ModelAttribute("user") UserSessionDetails loggedUser,
+	                                       @Valid @ModelAttribute("courseFilterForm")
+	                                           final CourseFilterForm courseFilterForm,
+	                                       final BindingResult errors) {
+
+		if (!loggedUser.hasAuthority("ADD_INSCRIPTION")
+				|| (loggedUser.getId() != docket && !loggedUser.hasAuthority("ADMIN"))) {
+			LOGGER.warn("User {} tried to add inscription for student {} and doesn't have ADD_INSCRIPTION authority [GET]", loggedUser.getDni(), docket);
+			return new ModelAndView(UNAUTHORIZED);
+		}
+
+		if (!model.containsAttribute("inscriptionForm")) {
+			model.addAttribute("inscriptionForm", new InscriptionForm());
+		}
+
+		final ModelAndView mav = new ModelAndView("courses");
+
+		final CourseFilter courseFilter = new CourseFilter.CourseFilterBuilder().
+				id(courseFilterForm.getId()).keyword(courseFilterForm.getName()).build();
+
+		final List<Course> courses = loadAvailableInscriptionsForDocketByFilter(docket, mav, errors, courseFilter);
+		if (courses == null) {
+			return NOT_FOUND_MAV;
+		}
+
+		final Student student = studentService.getByDocket(docket);
+		if (student == null) {
+			return NOT_FOUND_MAV;
+		}
+
+		mav.addObject("student", student);
+		mav.addObject("section2", "inscription");
+		mav.addObject("courseFilterFormAction", "/students/" + docket + "/inscription");
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Define a constant instead of duplicating this literal "/students/" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1192)